### PR TITLE
squeeze output map

### PR DESCRIPTION
**Pull request by alexriedel1**
# Description

It can happen for some models, that the output anomaly map can have an extra dimension [8,1,256,256] whereas the ground truth mask has not. This causes an error.
To fix this, the squeeze function is added before calculating pixel metrix.

Also this PR makes a change on how to read augmentation config from the config file to make image size HPO sweeps more accesible. 

This PR uses an ImportWarning instead of an ImportError for missing OpenVINO install. Otherwise without OpenVINO installed anomalib will crash.

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which refactors the code base)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x ] My code follows the [pre-commit style and check guidelines](https://github.com/openvinotoolkit/anomalib/blob/main/CONTRIBUTING.md) of this project.
- [x ] I have performed a self-review of my code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes
- [ ] I have added a summary of my changes to the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) (not for minor changes, docs and tests).

### CODE REVIEW

File: src/anomalib/models/efficientad/torch_model.py [hunk 1]
```diff
@@ -123,7 +123,6 @@ class Decoder(nn.Module):
     def __init__(self, out_channels, padding, img_size, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.img_size = img_size
-        self.last_upsample = 64 if padding else 56
```
Comment (djdameln) [inline src/anomalib/models/efficientad/torch_model.py:126]: Was this commited intentionally?

Comment (alexriedel1) [reply src/anomalib/models/efficientad/torch_model.py:126]: yes this line is not used as the same variable is (correctly) defined one line after..

**Review by samet-akcay — APPROVED** [review]
Thanks @alexriedel1
